kube/services/gdcapi/gdcapi-deploy.yaml

```diff
@@ -1,9 +1,9 @@
-apiVersion: extensions/v1beta1 
+apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: gdcapi-deployment
 spec:
-  replicas: 1 
+  replicas: 2
```

Comment:
Can we specify a surge instead? https://tachingchen.com/blog/Kubernetes-Rolling-Update-with-Deployment/